### PR TITLE
Records the decision to use S3

### PR DESCRIPTION
**Pull request by tosbourn**
Adds the tech decision log to use S3.

tldr; Hackney already use S3 and Rails has a long history of tooling that can talk to S3.

### CODE REVIEW

**Comment by tosbourn** [comment]
The PR is made up of 2 commits, which seems pernicity but I wanted a separation between the content and the act of acceptance.

Happy to combine into one though if we prefer.

File: doc/architecture/decisions/0013-use-s3.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+There is no user need to be able to edit an image once uploaded, as such the S3
+buckets will not need to have versioning enabled.
+
+## Decision
+
+Since there is an established pattern of using S3 for asset storage, we have
+decided to continue this pattern and use S3 for this project.
+
+We will be requested two buckets are made, one for the staging environment and
```
Comment (CristinaRO) [inline doc/architecture/decisions/0013-use-s3.md]: I'm a little confused at the phrasing here. "We will be requested", or "We will be requesting"?

Comment (tosbourn) [reply doc/architecture/decisions/0013-use-s3.md]: D'oh, thanks! Will address.

Comment (tosbourn) [reply doc/architecture/decisions/0013-use-s3.md]: @CristinaRO I've rewritten to use the correct tense

**Pull request merged**